package.json

```diff
@@ -33,23 +33,22 @@
     "babel-core": "7.0.0-bridge.0"
   },
   "dependencies": {
-    "@babel/core": "^7.1.2",
     "@sindresorhus/slugify": "^0.6.0",
-    "@storybook/react": "^4.0.0",
```

Comment:
(Silvi on Travis' laptop wishing he was watching braveheart on this flight to london but isn't baller enough to not work when it comes calling)

Reply:
(Travis on Travis' latop) Watching Dodgeball cause it's amazing tho def can't argue with Braveheart!